src/module.c

```diff
@@ -11016,7 +11527,21 @@ NULL
             argv = &c->argv[3];
         }
 
-        if (moduleLoad(c->argv[2]->ptr,(void **)argv,argc) == C_OK)
+        if (moduleLoad(c->argv[2]->ptr,(void **)argv,argc, 0) == C_OK)
+            addReply(c,shared.ok);
+        else
+            addReplyError(c,
+                "Error loading the extension. Please check the server logs.");
+    } else if (!strcasecmp(subcmd,"loadex") && c->argc >= 3) {
+        robj **argv = NULL;
+        int argc = 0;
+
+        if (c->argc > 3) {
+            argc = c->argc - 3;
+            argv = &c->argv[3];
+        }
+
+        if (moduleLoad(c->argv[2]->ptr,(void **)argv,argc, 1) == C_OK)
```

Comment:
nit pick, we're inconsistent about adding spaces, but let's at least be consistent within the line 8-)
```suggestion
        if (moduleLoad(c->argv[2]->ptr, (void **)argv, argc, 1) == C_OK)
```